hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/src/test/java/org/apache/hadoop/yarn/server/resourcemanager/scheduler/capacity/TestCapacitySchedulerConfiguration.java

```diff
@@ -107,6 +110,40 @@ public void testDefaultSubmitACLForRootAllAllowed() {
     assertTrue(acl.isAllAllowed());
   }
 
+  /**
+   * dfs.nfs.exports.allowed.hosts
+   * prop is deprecated, now we use
+   * nfs.exports.allowed.hosts
+   * instead
+   */
+  @Test
+  public void testDeprecationFeatureWorks() {
```

Comment:
How does the deprecation feature related to this PR?

Reply:
When we set a deprecated properite, there is some logic under the hood what keeps the old value and add an alias to the properties object, and we have to ensure the alias add also present in the tree.

The unset unfortunately not works fine, but i created a ticket for that
https://issues.apache.org/jira/browse/YARN-11348